schemas/proto/foxglove/LocationFix.proto

```diff
@@ -16,18 +18,24 @@ message LocationFix {
 
     KNOWN = 3;
   }
+  // Frame for the sensor. Latitude and longitude readings are at the origin of the frame.
+  string frame_id = 1;
+
+  // Timestamp of the message
+  google.protobuf.Timestamp timestamp = 2;
+
   // Latitude in degrees
-  double latitude = 1;
+  double latitude = 3;
```

Comment:
@jtbandes Have we done anything like this yet?

@amacneil I think we roughly treat these schemas as a snapshot set of examples rather than "use these exactly for all possible use-cases you have". Maybe we should call out this nuance in protobuf more explicitly but I've never known this repo to have a goal of keeping things backwards compatible.

Reply:
Yeah we have, search for `protobufFieldNumber`

Reply:
> I think we roughly treat these schemas as a snapshot set of examples rather than "use these exactly for all possible use-cases you have". Maybe we should call out this nuance in protobuf more explicitly but I've never known this repo to have a goal of keeping things backwards compatible.

Yeah agreed I don't think we explicitly make this promise. But it does seem a bit weird from a user's perspective for us to unnecessarily ship backwards-incompatible changes to the schema. So maybe we should make that promise going forward unless there is a particular change where that would be impossible. In this case looks like we just need to explicitly specify the numbers.

Reply:
Thanks for the tip on `protobufFieldNumber` I've added that to `LocationFix` to keep the existing field numbering unchanged.